doc/Caliptra.md

```diff
@@ -1368,20 +1369,25 @@ This section describes Caliptra error reporting and handling.
 
 | | Fatal Errors | Non-fatal Errors |
 | :- | - | - |
-| HW | <p>- ICCM, DCCM SRAM ECC</p><p>- Second WD timer expiry (First one triggers an NMI to FW to correct the issue and clear the interrupt status bit. If the WD expires again, then it gets escalated to FATAL error</p> | <p>- Mailbox SRAM ECC (except initial FW load)</p><p>- Mailbox incorrect protocol/commands</p><p>- Crypto processing errors</p><p> - FW requesting wrong key slot (eg. PCR key slot)</p><p>- Outside the AHB-lite decoding range access</p> |
-| FW | <p>- Boot-time FW authentication failures</p><p>- FW triggered FATAL errors. Examples: ICCM/DCCM misaligned access (potentially in the except subroutine); AHB access hangs that is triggered through WD timer expiry; stack overflow etc. Refer to the table below</p> | <p>- Mailbox API failures</p><p>- First WD timer expiry</p> |
+| HW | <p>- ICCM, DCCM SRAM ECC</p><p>- Second WD timer expiry (First one triggers an NMI to FW to correct the issue and clear the interrupt status bit. If the WD expires again, then it gets escalated to FATAL error</p> | <p>- Mailbox SRAM ECC (except initial FW load)</p><p>- Mailbox incorrect protocol/commands. E.g. Incorrect access ordering; access without Lock.</p> |
+| FW | <p>- Boot-time FW authentication failures</p><p>- FW triggered FATAL errors. Examples: ICCM/DCCM misaligned access (potentially in the except subroutine); AHB access hangs that is triggered through WD timer expiry; AHB access outside the decoding range; stack overflow etc. Refer to the table below</p> | <p>- Mailbox API failures</p><p>- First WD timer expiry</p><p>- Crypto processing errors</p> |
 
 **Fatal errors**
 
-- Fatal errors will log the FATAL ERROR reasons into an arch register that is RW from the external world. This register must be sticky (as in reset is on powergood)
-- Caliptra will signal this using a CALIPTRA\_FATAL\_ERROR wire
-	- SOCs must hook this into their SOC error handling logic
+- Fatal errors will log the FATAL ERROR reasons into an arch register that is RW from the external world. This register must be sticky (as in reset is on powergood).
+- This register may be cleared at any time via register write (W1C).
+- Caliptra will signal this using a cptra\_error\_fatal wire.
+	- SOCs must hook this into their SOC error handling logic.
+    - Error Mask registers (writable only by Caliptra uC) may be used to prevent error signal assertion per-event. Mask registers only impact interrupts when set prior to the error occurrence.
+    - cptra\_error\_fatal will remain asserted until Caliptra system is reset. Note that, although the HW FATAL ERROR register may be cleared at any time, a reset is still required to clear the interrupt.
```

Comment:
could you specify which reset? Also what is "Caliptra system", should this be "Caliptra core"?

Reply:
Not sure if this should be Caliptra Core or Caliptra Subsystem, since these are two different instantiation profiles. @bharatpillilli any thoughts?

Reply:
Per offline discussion, Subsystem reset is not required. Text updated to simply refer to "Caliptra" instead of "Caliptra system".